src/documents/Invokers.md

```diff
@@ -144,12 +134,8 @@ __Example:__
 fluid.defaults("xyz.widget", {
     ...
     invokers: {
-        // regular invokers:
         addVal: "xyz.widget.add({that}.staticVal, {arguments}.0)",
-        subtractVal: "{parent}.subtract({arguments}.0, {that}.staticVal)",
-        // dynamic invokers:
```

Comment:
Yes, and added to list of API changes - I'll push an update once I can figure out how to get the docpad generator to work....